logdetective/prompts-summary-only.yml

```diff
@@ -0,0 +1,9 @@
+# This file is intended for customization of prompts
```

Comment:
This one won't work. You need the `{}` for replacement field.

Reply:
??? I do not want to replace anything.

Reply:
The problem is how the prompt is meant to be used.
This prompt eventually reaches https://github.com/fedora-copr/logdetective/blob/main/logdetective/utils.py#L129 where it is used with `format` method.

If there is no `{}` replacement field. [1] There is no place for insertion of formatted log.

I realize this should have been specified from the start. But now it's at least in readme.[2]

[1] https://docs.python.org/3/library/string.html#format-string-syntax 
[2] https://github.com/fedora-copr/logdetective/tree/main?tab=readme-ov-file#system-prompts